source/resources/js/app/views/entry.js

```diff
@@ -164,7 +178,8 @@ export default Backbone.View.extend({
     removeEntry: function () {
         confirmDialog(
             `Delete ${this.model.get("title")}?`,
-            `Are you sure you want to delete this entry?`,
+            `Are you sure you want to delete this entry? This cannot be undone.`,
```

Comment:
I had removed `This cannot be undone part` since the entries will be moved to trash now.